prac_08/taxi.py

```diff
@@ -0,0 +1,37 @@
+"""
+CP1404 Practical
+Car class
+"""
+
+from prac_08.car import Car
+
+
+class Taxi(Car):
+    """Specialised version of a Car that includes fare costs."""
+    price_per_km = 1.23
+
+    def __init__(self, name, fuel, price_per_km):
+        """Initialise a Taxi instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.price_per_km = price_per_km
+        self.current_fare_distance = 0
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return "{}, {}km on current fare, ${:.2f}/km".format(super().__str__(),
+                                                             self.current_fare_distance,
+                                                             self.price_per_km)
+
+    def get_fare(self):
+        """Return the price for the taxi trip."""
+        return self.price_per_km * self.current_fare_distance
```

Comment:
I believe that the question for this program asks you to use the 'round()' built-in